widget/tree.go

```diff
@@ -68,6 +71,11 @@ func NewTreeWithStrings(data map[string][]string) (t *Tree) {
 	return
 }
 
+// ClearSelection clears the current selection.
```

Comment:
ClearSelection and SetSelection are here but no Selection() method

Reply:
Ooops forgot that one. Added now :)